Assets/Scripts/Tools/NodeTool.cs

```diff
@@ -10,7 +10,7 @@ public class NodeTool : MonoBehaviour, ITool, IToolChangeObserver
     GraphService _graphService;
     ToolPanelController _toolPanelController;
     private Camera _attachedCamera;
-    private bool _shouldGhostNodeBeActive = false;
+    bool _lastCanInteractWithResult = false;
     private GameObject _ghostNode;
     private const float c_maxInteractDistance = 10f;
     private const float c_addNodeDistance = 4f;
```

Comment:
why this style for naming consts?

Reply:
I am pretty sure I mentioned this style when we were discussing it.
I am just used to it and have not found a nicer style for them. The default we have now is PascalCase but that's just confusing with properties. Const should have some indication that they are indeed consts so a normal private _ is not ideal as well. c_ may not look nice, but does the job and it's nice in the editor.